contracts/Safe.sol

```diff
@@ -267,12 +267,12 @@ contract Safe is
     /**
      * @inheritdoc ISafe
      */
-    function checkSignatures(bytes32 dataHash, bytes memory signatures) public view override {
+    function checkSignatures(address executor, bytes32 dataHash, bytes memory signatures) public view override {
```

Comment:
This change would break backwards compatibility, right? 

https://github.com/safe-global/safe-smart-account/blob/main/contracts/Safe.sol#L270

Reply:
No - this is a new method added to 1.5.0. The backwards compatible function is still there:

https://github.com/safe-global/safe-smart-account/blob/e9a75acf795c70949bde934fcad51d4711dad066/contracts/Safe.sol#L334-L347